### PR TITLE
Adding e2e tests

### PR DESCRIPTION
**Pull request by christian-bromann**
This test verifies if Runme Wasm is properly loaded by VS Code and an execution is triggered correctly.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ jobs:
         with:
           node-version: 18
       - name: 📦 Install Dependencies
-        run: yarn
+        # use NPM here as Yarn as issues resolving WebdriverIO dependencies
+        run: npm install
```
Comment (christian-bromann) [inline .github/workflows/test.yml:17]: `@wdio/runner": "^8.0.0-alpha.558` depends on `"expect-webdriverio": "^4.0.0-alpha.3"` but it installed `v4.0.0` which was a [broken release](https://www.npmjs.com/package/expect-webdriverio/v/4.0.0) and I deprecated.

**Comment by sourishkrout** [comment]
Reviewing now

File: .github/workflows/test.yml (hunk 1 above)
Comment (sourishkrout) [reply .github/workflows/test.yml:17]: Perhaps we should just track a ticket to migrate from `yarn` to `npm` in dev too?

**Review by sourishkrout — APPROVED** [review]
Didn't work locally with `yarn` but once I installed deps with `npm` it ✅ .

**Pull request merged**